ArduCopter/Copter.cpp

```diff
@@ -309,6 +309,21 @@ bool Copter::set_target_velocity_NED(const Vector3f& vel_ned)
     return true;
 }
 
+bool Copter::set_target_angle(const float roll_deg, const float pitch_deg, const float yaw_deg, const float climb_rate_cms, const bool use_yaw_rate, const float yaw_rate_rads)
```

Comment:
As a small side note, I'm really not a fan of adding "const" in front of arguments.  I know there are varying opinions on this but I'm personally not a fan because it makes the line much longer and harder to read 'cuz more words to ignore before getting to what matters.  This isn't a blocker though.

Reply:
fixed